robusta/build.rs

```diff
@@ -1,12 +1,18 @@
+use specta;
 use std::env;
+use std::path::PathBuf;
 use std::process::Command;
 
+const FRONTEND_DIR: &str = "liberica";
```

Comment:
I'm really not sure it makes sense to put this into a constant but it is fine I guess

Reply:
This was actually an artifact of an old change, which I later removed.